src/main.ts

```diff
@@ -53,15 +55,22 @@ export async function initRemix({
     app.whenReady(),
   ])
 
+  let lastBuildTime: number = 0;
+  let buildPath = typeof serverBuildOption === "string" ? join(serverBuildOption, 'index.js') : join(process.cwd(), 'desktop/build.js')
```

Comment:
- This isn't stated anywhere explicitly, but remix-electron doesn't support hot requires unless a build path is given. Trying to guess the path can go wrong if the user changed it in their config, so it should probably be undefined if it's not a string
- The server build option can be anything require-able, and it might not always be an index.js file. `require.resolve` would be a more reliable way to find the build output.
```suggestion
  const buildPath = typeof serverBuildOption === "string"
  	? require.resolve(serverBuildOption)
  	: undefined
```